ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -46,8 +46,14 @@ PathogenicityLink.propTypes = {
 }
 
 
+const clinvarUrl = (clinvar) => {
+  const baseUrl = 'http://www.ncbi.nlm.nih.gov/clinvar'
+  const variantPath = clinvar.variantId ? `/variation/${clinvar.variantId}` : `?term=${clinvar.alleleId}[alleleid]`
```

Comment:
you want to default to alleleid (unless it's missing for some reason)

Reply:
(the allele id is more specific than variant id)

Reply:
got it